spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -13,7 +13,10 @@ describe("kong start/stop #" .. strategy, function()
   end)
   after_each(function()
     helpers.kill_all()
+    ngx.update_time()
+    ngx.sleep(2)
```

Comment:
replace extra sleep with for example remove the events socket file, instead of adding more sleep